### PR TITLE
[WIP] Adding query changes to run on Dask-CPU

### PR DESCRIPTION
**Pull request by DaceT**
Made changes to the GPU-BDB queries so that it can run with Dask-CPU.

### CODE REVIEW

**Review by randerzander — CHANGES_REQUESTED** [review]
Thanks for opening this so others can see it!

Some initial thoughts, but most of the review should likely wait until the dask-sql PR merges and this PR is synced w/ it

File: gpu_bdb/benchmark_runner.py [hunk 1]
```diff
@@ -22,14 +22,16 @@ def load_query(qnum, fn):
 
 
 dask_qnums = [str(i).zfill(2) for i in range(1, 31)]
-bsql_qnums = [str(i).zfill(2) for i in range(1, 31)]
+dask_qnums = []
```
[the remark this answers is not in the document]
Comment (DaceT) [reply gpu_bdb/benchmark_runner.py:25]: ok, will do!

File: conda/rapids-gpu-bdb.yml [hunk 2]
```diff
@@ -5,17 +5,17 @@ channels:
...
-  - cudatoolkit=11.0
+  - cudatoolkit=11.2
   - cudf
   - rmm
   - dask-cuda
   - dask-cudf
   - cuml
   - dask
   - distributed
+  - maven
```
[the remark this answers is not in the document]
Comment (ayushdg) [reply conda/rapids-gpu-bdb.yml:16]: I think @ChrisJar PR can add dask-sql as a dependency which would remove requiring maven as a pre-requisite.

File: gpu_bdb/build/lib/bdb_tools/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
Comment (ayushdg) [inline gpu_bdb/build/lib/bdb_tools/__init__.py:1]: Add all files within `build` and it's subfolders to gitignore and remove all the installed lib files.

**Review by ayushdg — CHANGES_REQUESTED** [review]
Added comments inline. 

In addition to that it might make sense to update this Pr with updates made to #224 and a more comprehensive review can be started once that get's merged in

**Pull request closed without merging**